docs/fleet/fleet.md

```diff
@@ -10,21 +10,37 @@ Fleet provides a centralized view of all the [Sematext Agents](https://sematext.
 - Version: Installed version of the Sematext Agent and whether important updates are available. If they are, you can update the agent by just clicking the link and following the instructions. You can also find those instructions [here](https://sematext.com/docs/monitoring/spm-faq/#agent-updating).
-- Status: Indicates the status of the Sematext Agent. Possible statuses are:
-    - Running: Everything is nominal.
-    - Stopped: The agent has been stopped by the user. You can [manually restart](https://sematext.com/docs/agents/sematext-agent/starting-stopping) the Sematext Agent.
-    - Failed: The agent is reporting errors. You can see them in the Agent Information Panel, in the Errors tab.
-    - Dead: The agent has stopped due to an error. You can see the errors in the Agent Information Panel, in the Errors tab, and you can [manually restart](https://sematext.com/docs/agents/sematext-agent/starting-stopping) the Sematext Agent.
-    - Pending or Unknown: the agent is not sending data to Sematext Cloud. Check if the agent is still running on your system. If it is, there may be connection issues.
-- Started: Time elapsed since the Sematext Agent began sending data to Sematext Cloud.
-- Infra App: the Infra App which is receiving the data from the Sematext Agent. Clicking on the App name will navigate you to the overview of the corresponding App in the [Infrastructure](https://sematext.com/docs/monitoring/infrastructure/) section of Sematext Cloud.
+- **Status**: Indicates the status of the Sematext Agent.
+- **Started**: Time elapsed since the Sematext Agent began sending data to Sematext Cloud.
+- **Infra App**: The Infra App which is receiving data from the Sematext Agent. Clicking on the App name takes you to the overview of the corresponding App in the [Infrastructure](https://sematext.com/docs/monitoring/infrastructure/) screen.
+
+## Agent Versions & Updating
+The version of Sematext Agent tells you how up to date each agent instance is.  You can always check the [Sematext Agent Release Notes](https://sematext.com/docs/agents/sematext-agent/releasenotes/) to see all changes in each agent version, including any breaking changes, which are always called out explicitly.  When any instance of Sematext Agent in your infrastructure is out of date it will be marked as outdated.  You can see all outdated agent instances by clicking on the outdated badge.
+
+<screenshot showing the outdated badge>
+
+The agent update instructions are shown in the [Upgrade Instructions](https://sematext.com/docs/fleet/fleet/#upgrade-instructions) tab.
+
+## Agent Status
+At any one time an agent is in one of the following statuses:
+
+- **Running**: Everything is nominal.
+- **Stopped**: The agent has been stopped by the user. You can [manually restart](https://sematext.com/docs/agents/sematext-agent/starting-stopping) the Sematext Agent.
```

Comment:
@cpipilas what if something kills it (i.e., it wasn't stopped by a user) - won't the agent show up a stopped then, too?